component/main.jsonnet

```
@@ -1,10 +1,17 @@
 // main template for control-api
+local controlApi = import 'lib/control-api.libjsonnet';
 local kap = import 'lib/kapitan.libjsonnet';
 local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.control_api;
 
+local zones = [
+  controlApi.Zone(name) + params.zones[name]
+  for name in std.filter(function(it) it != null, std.objectFields(params.zones))
+];
```

This should probably be

```suggestion
local zones = [
  controlApi.Zone(name) + params.zones[name]
  for name in std.objectFields(params.zones)
  if params.zones[name] != null
];
```
The current implementation would filter out `null` _keys_, which probably wasn't the intention.